libs/common/src/utils/queues.ts

```diff
@@ -17,6 +17,11 @@ export namespace QueueConstants {
    * Name of the queue that has all the jobs and items that needs to run periodically or check their status
    */
   export const STATUS_QUEUE_NAME = 'statusQueue';
+
+  /**
+   * Name of the queue that has all the transaction receipts
+   */
+  export const TRANSACTION_RECEIPT_QUEUE_NAME = 'transactionReceiptQueue';
```

Comment:
I was thinking maybe TRANSACTION_RECEIPT_QUEUE would the same  STATUS_QUEUE but it's fine to split them for now and we can refactor later in case if it made sense to do so.

Reply:
Yes I was a bit conflicted on it but can check laters